<launch>
	<node pkg="lab6" type="safety.py" name="Safety_Test" output="screen">
		<param name="scan_topic" value="/scan"/>
    <param name="drive_topic" value="/vesc/high_level/ackermann_cmd_mux/output"/>
    <param name="safety_topic" value="/vesc/low_level/ackermann_cmd_mux/input/safety"/>
    <param name="stop_dist_factor" value="0.1"/>
    <param name="stop_dist" value="0.5"/>
	</node>
</launch>
